### PR TITLE
bug fix to pass logger object in tools script

### PR DESCRIPTION
**Pull request by gulshan02**
Explain what this PR does.

### CODE REVIEW

File: numalogic/udfs/tools.py [hunk 1]
```diff
@@ -273,7 +273,7 @@ def __fetch_ts(self, uuid: str, key: str) -> _DedupMetadata:
                 msg_train_records=_msg_train_records,
             )
 
-    def ack_insufficient_data(self, key: KEYS, uuid: str, train_records: int) -> bool:
+    def ack_insufficient_data(self, logger, key: KEYS, uuid: str, train_records: int) -> bool:
```
Comment (s0nicboOm) [inline numalogic/udfs/tools.py:276]: can we have the typing? 
Also, can we have that Optional parameter? What do you think?

**Review by s0nicboOm — APPROVED** [review]
minor comment otherwise lgtm

File: numalogic/udfs/tools.py (hunk 1 above)
Comment (ab93) [reply numalogic/udfs/tools.py:276]: I don't think logger should come in as an argument. Can you figure out some other way @gulshan02 ?

**Review by ab93 — CHANGES_REQUESTED** [review]

File: numalogic/udfs/tools.py (hunk 1 above)
Comment (gulshan02) [reply numalogic/udfs/tools.py]: done

**Review by ab93 — APPROVED** [review]

**Pull request merged**